docs/docs/invalidtickers.md

```diff
@@ -0,0 +1,202 @@
+# Test Case Document
```

Comment:
Might want to change the name of this file to be `testcase_invalidtickers.md` to be more consistent